theme/horizontalrule.css

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+.ck-content .ck-horizontal-rule {
+	/* Handle proper `<hr>` display next to elements with `float` property, e.g. side image case. */
+	overflow: hidden;
```

Comment:
Is it ok that this style is only here? What about content produced by CKE5? Will hrs be trimmed there?

Reply:
cc @dkonopka

Reply:
It looks everything ok:

![Screen Shot 2019-09-05 at 09 45 01](https://user-images.githubusercontent.com/10219857/64321965-e474ac00-cfc1-11e9-942d-cd6bc238b51b.png)

Demo: https://jsfiddle.net/26qopj3s/